bitnami/contour/templates/default-backend/ingress.yaml

```diff
@@ -10,11 +10,7 @@ metadata:
     {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
     {{- end }}
   annotations:
-    {{- if .Values.ingressClass }}
-    kubernetes.io/ingress.class: {{ .Values.ingressClass }}
-    {{- else }}
-    kubernetes.io/ingress.class: contour
-    {{- end }}
+    kubernetes.io/ingress.class: {{ include "contour.ingressClassName" . }}
```

Comment:
I found extra field which uses `ingressClass` name while self-reviewing 🤔

Not only `contour` was used as default value on `default-backend`, but also contour [assumes `contour` as default ingress class name](https://github.com/projectcontour/contour/blob/5de4f21c92df9069c0fcb3998636da9f48db377d/internal/ingressclass/ingressclass.go#L23-L26). I think it would be better if we set default ingress class name explicitly - https://github.com/bitnami/charts/pull/7668/commits/ac129e427f07d394f73e5b870bfc1b2198516646, WDYT?